### PR TITLE
Simplify Wool

### PR DESCRIPTION
**Pull request by tenplus1**
Simplify wool mod to go with dye mod pull: https://github.com/minetest/minetest_game/pull/2186

### CODE REVIEW

File: mods/wool/init.lua [hunk 1]
```diff
@@ -35,13 +31,10 @@ for i = 1, #dyes do
...
-		recipe = {"group:dye," .. craft_color_group, "group:wool"},
+		recipe = {"group:dye,color_" .. name, "group:wool"},
 	}
 end
 
-
--- legacy
-
--- Backwards compatibility with jordach's 16-color wool mod
+-- legacy (Backwards compatibility with jordach's 16-color wool mod)
```
Comment (paramat) [inline mods/wool/init.lua]: 'Legacy' and no need for brackets.

Comment (paramat) [inline mods/wool/init.lua]: I prefer:
```
-- Legacy
-- Backwards compatibility with jordach's 16-color wool mod
```

**Comment by paramat** [comment]
Haven't tested but :+1:

**Comment by tenplus1** [comment]
@paramat - github wont allow me to edit and change this pull, if you could do it from your side that'd be great, otherwise I'm gonna have to delete and create new pull to do so.

**Comment by paramat** [comment]
Ok we'll do this on merge.

**Comment by paramat** [comment]
Replaced by #2202

**Pull request closed without merging**